docs/source/EccFramework.rst

```diff
@@ -0,0 +1,154 @@
+ECC Framework
+======================================================================================================
+
+The QECC library offers means for automatic implementation and evaluation of error-correcting codes for quantum
+computing. More precisely, the library allows to automatically apply different error correction schemes to quantum
+circuits provided as OpenQASM files or Qiskit :code:QuantumCircuit objects. The "protected" quantum circuits can then be exported again in the form of OpenQASM files or can be directly used for noise-aware quantum circuit simulation. For the latter case, we also provide a wrapper script which makes use of the provided framework to apply error correction schemes to circuits and directly simulate those circuits using qiskit.
+
+Usage
+#####
+
+Having the Python module installed, error correction can be applied to a quantum circuit using :func:~mqt.qecc.apply_ecc, like so
```

Comment:
This, e.g., is also missing backticks. Please carefully check if that is enough to properly render the reference.

Reply:
I changed it so: :func:`~mqt.qecc.apply_ecc`